### PR TITLE
CLOUDP-143633: Automate Openshift release

### PR DESCRIPTION
**Pull request by helderjs**
### All Submissions:

* [x] Have you signed our [CLA](https://www.mongodb.com/legal/contributor-agreement)?

### CODE REVIEW

File: .github/workflows/release-openshift.yaml [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+          preflight check container "${IMG_SHA}" \
+            --submit \
+            --pyxis-api-token="${RH_CERTIFICATION_PYXIS_API_TOKEN}" \
+            --certification-project-id="${RH_CERTIFICATION_OSPID}" \
+            --docker-config=./authfile.json
+
+          # Replace image version with SHA256
+          value="${IMG_SHA}" yq e -i '.spec.install.spec.deployments[0].spec.template.spec.containers[0].image = env(value)' \
+            "${VERSION}"/manifests/mongodb-atlas-kubernetes.clusterserviceversion.yaml
+
```
Comment (igor-karpukhin) [inline .github/workflows/release-openshift.yaml:89]: There is one more place to replace the image tag with image SHA: `.metadata.annotations.containerImage`

Comment (igor-karpukhin) [reply .github/workflows/release-openshift.yaml:89]: Also, for certified operators please make sure that images have `quay.io/` prefix

Comment (helderjs) [reply .github/workflows/release-openshift.yaml]: Done!
Also, as we now support multi-arch, we don't use the manifest digest(sha256) as the version anymore.
We just refer to the actual version of the release

**Review by Sugar-pack — APPROVED** [review]
LGTM

File: .github/workflows/release-openshift.yaml [hunk 2]
```diff
@@ -0,0 +1,106 @@
...
+      - name: Configure certified release
+        if: ${{ matrix.certified }}
+        env:
+          VERSION: ${{ github.event.inputs.version }}
+          REPOSITORY: ${{ matrix.repository }}
+          IMAGE: quay.io/mongodb/mongodb-atlas-kubernetes-operator
+        run: |
+          # Reference to image
+          REPO_DIGEST=$(docker inspect --format='{{ index .RepoDigests 0}}' "${IMAGE}":"${VERSION}")
+          containerImage="quay.io/${REPO_DIGEST}"
```
Comment (igor-karpukhin) [inline .github/workflows/release-openshift.yaml:74]: @helderjs  It looks like the path to the image is missing, isn't it?

**Pull request merged**